### DISCUSSION TITLE
Help Needed

### DISCUSSION THREAD

**Discussion by fromthelifestream** [vincentneo/LosslessSwitcher#100] [Q&A] [answered]
Hey,

sorry if in the wrong place but I can't seem to get it to work. On a mac mini m2 pro. Using latest beta release. It just stays at 48khz and I can't seem to find any info on troubleshooting or what I could be doing wrong.

Would very much appreciate some guidance, thank you!

**Comment by vincentneo** [comment]
Hi @fromthelifestream, 

Yes this is the right place to ask! No worries about that.

A few questions to know the situation better:
1. Are you using LosslessSwitcher with the bit depth enabled or disabled? 
2. Which macOS version are you using?

**Reply by fromthelifestream** [reply]
1. Enabled.
2. Today's update - Ventura 13.4.1

**Reply by vincentneo** [reply]
Try disabling it. It's known to work better with bit depth detection disabled.

**Reply by fromthelifestream** [reply]
I've tried both, still stays at 48 but I'll disable it.

**Reply by MasterRahool** [reply]
in Selected Device, have you selected your DAC, or is it on default?

**Reply by fromthelifestream** [reply]
I'm using steven slate VSX systemwide so I have that selected but I've tried external headphones as well as default.

**Reply by vincentneo** [reply]
In the Audio MIDI Setup app, were you able to set your sample rate there?

**Reply by fromthelifestream** [reply]
I see you changed your comment to DAC now. I'm not using one, is that the problem then?

**Reply by vincentneo** [reply]
nah even if you're using the Mac's internal speakers, if it supports more than one sample rate setting, it should still switch fine.

**Reply by fromthelifestream** [reply]
I am able to change in audio midi on my external headphones but have no option on VSX systemwide. However no change in functionality when using external headphones and changing sample rate manually.

New to all of this so please excuse my ignorance.

**Reply by MasterRahool** [reply]
> nah even if you're using the Mac's internal speakers, if it supports more than one sample rate setting, it should still switch fine.


Macbook speakers have a drop menu to change *but* my display's internal speakers are locked at 48k, so just covering bases that his device might have 1 setting ;)

**Reply by vincentneo** [reply]
in that case:
The `steven slate VSX` (I assume it shows as a virtual device) cannot be used as a selected option.
Selecting external headphones should work? 

> New to all of this so please excuse my ignorance.

No worries about that! Please feel free to ask anything!

**Reply by vincentneo** [reply]
@MasterRahool ah I see! My Mac Studio shows 44.1 to 96khz so I guess newer Macs do support that

**Reply by fromthelifestream** [reply]
Oh, no it can be selected. Unsure which one I should pick but again I've tried both.

Much appreciated, thank you!

**Reply by fromthelifestream** [reply]
I have a mac studio display if that info helps.

**Reply by vincentneo** [reply, marked as answer]
Oh I mean that the VSX thing shouldn't be selected in LosslessSwitcher.

LosslessSwitcher works the same way as a user switching sample rates in MIDI setup app, just that it does it for you. If a user can't select, LosslessSwitcher can't select too.

Now since you've said you've tried both, I wonder if it has to do with some incompatibility with the VSX thing.

Can I trouble you to:
1. disable the VSX thing
2. select external headphones as the playback device

**Reply by fromthelifestream** [reply]
It did! Now that really bums me out because the audio quality boost from VSX is so high. I was able to get both to work if when I re-started and then re-selected external headphones in VSX.. however when the track changes the VSX gets disabled and I have to re-select external headphones and change the volume once for it to work..

Sorry if I'm confusing you, this is confusing me...

**Reply by vincentneo** [reply]
So the VSX incompatibility is the reason here? Or did you get both to cooperate?

The whole thing is pretty confusing indeed 😅

**Reply by fromthelifestream** [reply]
I get them to cooperate but when the track changes i get muted. or deafened I guess.. the sounds disappears. If i adjust volume it comes back but VSX is disabled. If I enable VSX and then select external headphones as well as adjust volume once, everything works.🤷‍♂️

**Reply by fromthelifestream** [reply]
(Which is a real bummer because I obviously can't do that whole song and dance, pun intented, every change of song.)

**Reply by vincentneo** [reply]
I see. Maybe I should write somewhere that this configuration doesn't work.

**Reply by fromthelifestream** [reply]
Well, it's not your problem ofc. I feel it definitely could work though since it's technically possible with the whole ritual I did. 

Either way I think VSX and AI software sound systems are probably where the futures heading. Lossless is obviously a game changer but having to pick between lossless and VSX just now, it wasn't even close. VSX is an order of magnitude improvement on audio quality compared to lossless. Pretty sure it's magic. 

Thanks for all the help!

**Reply by vincentneo** [reply]
Feel free to let me know if there's any other questions. Thank you!

**Reply by fromthelifestream** [reply]
Hey again!

Forgot to say that I got it to work fully by using the studio display as selected device in lossless and external headphones as selected output device in VSX. Probably good to know. Thanks again!

**Reply by vincentneo** [reply]
Thanks for letting me know! 

This is an interesting workaround, as I would have thought that would change the sample rate output for the studio display’s speakers rather than your headphones!

**Comment by park-sunkyu** [comment]
hi~
2025.04.24 After the mac mini OS Sequoia 15.4.1 update, lossesswitcher does not work. Is it just me? It is a mac mini M4 product.

**Reply by nezil** [reply]
There is a separate discussion about this. It's a known issue. Version 1.0 works, but it's kinda buggy and doesn't have the refinement of the newer versions that don't work.
